doc/architecture/reference_points_coordinate_systems.adoc

```diff
@@ -0,0 +1,17 @@
+= Coordinate systems and reference points
+
+OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.
+OSI uses three coordinate systems:
+
+World coordinate system::
+Coordinate system for all quantities which are part of ground truth.
+The world coordinate system is an inertial x/y/z-coordinate system.
```

Comment:
```suggestion
The world coordinate system is an inertial x/y/z-coordinate system.
The origin is the global reference point, which is determined by the environment simulation.
It may be derived from map data or other considerations.
A mapping to a geographic coordinate system can be provided optionally via a proj.4 string.
```

Reply:
https://opensimulationinterface.github.io/open-simulation-interface/structosi3_1_1GroundTruth.html#af5f78f308a00d74ece79410eb7913006

Reply:
Changed, please check whether this is correct.